src/Common/test-runtime/project.json

```diff
@@ -9,7 +9,10 @@
       "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0040",
       "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24728-02",
       "Microsoft.NETCore.TestHost": "1.2.0-beta-24728-02",
-      "Newtonsoft.Json": "9.0.1"
+      "Newtonsoft.Json": "9.0.1",
```

Comment:
The code coverage targets just use them out of the packages directory, actually, so it's not needed. It is kind of weird to have some of the things listed here just so they get restored, though. Any ideas?